src/sentry/incidents/endpoints/serializers/alert_rule_trigger_action.py

```diff
@@ -6,67 +6,75 @@
         if priority:
-            priority += " level"
+            return f"Send a {priority} Opsgenie notification to " + target_display
+        return "Send an Opsgenie notification to " + target_display
+    elif action_type == AlertRuleTriggerAction.Type.DISCORD.value:
+        if not target_display:
+            logger.info(
+                "discord.action.description.no.channel",
+                extra={"target_identifier": target_identifier},
+            )
+        return f"Send a Discord notification to {target_display}"
 
-        if action.type == action.Type.EMAIL.value:
-            if action.target:
-                if action.target_type == action.TargetType.USER.value:
-                    return "Send a notification to " + action.target.email
-                elif action.target_type == action.TargetType.TEAM.value:
-                    return "Send an email to members of #" + action.target.slug
-        elif action.type == action.Type.PAGERDUTY.value:
-            return f"Send a {priority} PagerDuty notification to " + action.target_display
-        elif action.type == action.Type.SLACK.value:
-            return "Send a Slack notification to " + action.target_display
-        elif action.type == action.Type.MSTEAMS.value:
-            return "Send a Microsoft Teams notification to " + action.target_display
-        elif action.type == action.Type.SENTRY_APP.value:
-            return "Send a notification via " + action.target_display
-        elif action.type == action.Type.OPSGENIE.value:
-            if priority:
-                return f"Send a {priority} Opsgenie notification to " + action.target_display
-            return "Send an Opsgenie notification to " + action.target_display
-        elif action.type == action.Type.DISCORD.value:
-            if not action.target_display:
-                logger.info(
-                    "discord.action.description.no.channel",
-                    extra={"target_identifier": action.target_identifier},
-                )
-            return f"Send a Discord notification to {action.target_display}"
 
-    def get_identifier_from_action(self, action):
-        if action.type in [
-            AlertRuleTriggerAction.Type.PAGERDUTY.value,
-            AlertRuleTriggerAction.Type.SENTRY_APP.value,
-        ]:
-            return int(action.target_identifier)
-        if action.type == AlertRuleTriggerAction.Type.OPSGENIE.value:
-            # return team ID: opsgenie team IDs are strings
-            return action.target_identifier
-        # if an input_channel_id is provided, we flip these to display properly
-        return (
-            action.target_display if action.target_display is not None else action.target_identifier
-        )
+def get_identifier_from_action(action_type, target_identifier, target_display=None):
+    if action_type in [
+        AlertRuleTriggerAction.Type.PAGERDUTY.value,
+        AlertRuleTriggerAction.Type.SENTRY_APP.value,
```

Comment:
Does this also need to include like `Action.Type.PAGERDUTY.value` to work for both action types?